src/styles/calendar.scss

```diff
@@ -0,0 +1,55 @@
+
+@import "./variables.scss";
+
+.calendar {
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  max-width: $container-width;
+  padding: 30px;
+  gap: 1px;
+
+  &__block {
+    height: 98px;
+    width: 98px;
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: background-color 0.5s, transform 0.5s;
+
+    &::before {
+      line-height: 100px;
+      display: flex;
+      justify-content: center;
+    }
```

Comment:
I am not sure wether you need it, you might add display: flex, justify-content and align-items to calendar__block instead

Reply:
Yeah, same result but looks better.